public/mockServiceWorker.js

```diff
@@ -0,0 +1,303 @@
+/* eslint-disable */
+/* tslint:disable */
+
+/**
+ * Mock Service Worker (1.2.1).
+ * @see https://github.com/mswjs/msw
+ * - Please do NOT modify this file.
+ * - Please do NOT serve this file on production.
```

Comment:
need to check how to not serve this in production then

Reply:
can we check this issue, we can follow this https://github.com/Real-Dev-Squad/website-calendar/pull/150 ?